examples/source/1.components/amp-script/index.html

```diff
@@ -219,6 +219,45 @@
       </script>
     </div>
 
+    <!--
+      ## Data source for `amp-list`
+      An `<amp-script>` function can serve as the data source for an `<amp-list>`.
+        - Use `exportFunction()` to make the function visible to the `<amp-list>`.
+        - Specify the script and the function in the `src` attribute of the `<amp-list>`. Use the form `src="amp-script:{scriptID}:functionName"`, where `{scriptID}` is the `id` of the `<amp-script>` and `{functionName}` is the name of the exported function.
+        - You can use the `nodom` attribute in the `<amp-script>` to indicate that the `<amp-script>` won't need a DOM. This improves performance.
```

Comment:
neat! 
if possible to do so succinctly, it would great to mention in what way performance is improved

Reply:
I got this from @samouri , so I'd like to ask him to comment here!

* I know that when `nodom` is used, `amp-script` loads a smaller bundle ~ 2K instead of 15K. I believe this is because it won't need any of the logic WorkerDOM uses to handle the DOM API. Does that smaller bundle not include WorkerDOM at all?
* I imagine it also might mean that the JS that builds the virtual DOM doesn't need to execute (and might not even be loaded), saving time... perhaps even more JS doesn't need to execute.

Anything else?

Reply:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-script/0.1/amp-script.js#L324

Reply:
> Does that smaller bundle not include WorkerDOM at all?

Pretty accurate! It technically includes the `worker-dom` library still, but it removes the vdom implementation

Reply:
Ok! Put in a bit about no vdom.